service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/SendDocumentTaskExecutor.java

```diff
@@ -74,7 +74,7 @@ public void execute(SendDocumentTaskDefinition taskDefinition) {
                 var messageId = randomIdGeneratorService.createNewId();
                 var filename = mainMessageId + "_" + i + MESSAGE_ATTACHMENT_EXTENSION;
                 var chunkPayload = generateChunkPayload(taskDefinition, messageId, filename);
-                var chunkedOutboundMessage = createChunkOutboundMessage(chunkPayload, chunk, contentType);
+                var chunkedOutboundMessage = createChunkOutboundMessage(chunkPayload, chunk, "application/octet-stream");
```

Comment:
I'm suspicious that no tests failed when making this change. Will introduce a unit test which covers this behaviour.

Reply:
This was resolved in #517